packages/validation/src/databases.schema.ts

```diff
@@ -1,4 +1,5 @@
 import { array, object, string } from 'yup';
+import { IP_ERROR_MESSAGE, validateIP } from '../lib/firewalls.schema';
```

Comment:
For some reason, importing lib schemas inside a schema causes a compile error so we should import from src instead
![image](https://user-images.githubusercontent.com/14323019/149194326-7a5d9138-87bb-4b4c-bd84-95172ecf338c.png)

```suggestion
import { IP_ERROR_MESSAGE, validateIP } from './firewalls.schema';
```